packages/radix-ui-themes/src/components/base-radio.css

```diff
@@ -15,6 +15,10 @@
   border-radius: var(--skeleton-radius);
   --skeleton-radius-override: 100%;
 
+  &:where(:disabled, [data-disabled]) {
+    cursor: var(--cursor-disabled);
+  }
```

Comment:
- The `::before` pseudo-element is currently used to render the checkbox and radio chrome, and the root targets the clickable area, which may be a bit larger that the visible element in certain layouts
- `cursor` styles is Safari don't apply to pseudo-elements (seems to be a relatively well-known issue)
- Ideally, we’d work around that using a separate `span` element for the chrome and put the `cursor` style there too
- However, we can’t do the above with the `Radio` component which renders a regular `input`, so I decided to put the cursor style back onto the entire clickable area given that it isn't much larger in reasonable layout situations